.storybook/preview.js

```diff
@@ -32,7 +32,7 @@ export const decorators = [
 ];
 
 export const parameters = {
-  actions: { argTypesRegex: "^on[A-Z].*" },
+  // actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
Can we have a comment here for why this is removed? Also, please remove commented lines.

Reply:
With Storybook 8.0, argTypesRegex is removed so a different config is recommended to make play functions work. But i cross-checked, we don't need it as there are no play functions being used